kamailio/default.cfg

```diff
@@ -380,17 +395,20 @@ route[CHECK_RETRANS]
     }
 }
 
+
+modparam("htable", "htable", "insane=>size=32;autoexpire=120;initval=0;updateexpire=1;")
+
 route[SANITY_CHECK]
 {
-    ## CVE-2018-14767
-    if($(hdr(To)[1]) != $null) {
-        xlog("second To header not null - dropping message");
+    if($sht(insane=>$si:$sp) > 3) {
```

Comment:
Several security issues is fixed on latest `kamailio:5.1.6`. `5.1` branch is tested by me.
How about to migrate config to `5.2` branch? Now kamailil/kazoo config is not compatible with `kamailio:5.2` branch.

Reply:
@sergey-safarov we can't use `5.2` due to changes in `dispatcher` module (still testing)